### ISSUE TITLE
Duplicate paragraph of text in understanding of 3.3.8

### ISSUE THREAD

**Issue by gjccopinga** [w3c/wcag#3122]
In the understanding document of success criterion 3.3.8 there is a duplicate paragraph of text. The final paragraph in the Intent section is a duplicate. The same paragraph can be found three paragraphs above this one. It starts wit the text "Another factor that can contribute to cognitive load is hiding characters when typing."  One of there paragraphs need to be removed.
See: https://w3c.github.io/wcag/understanding/accessible-authentication.html

**Comment by alastc** [comment]
Thanks, it also looks like the headings weren't come through due to a lack of `section` tags, so I updated that as well.
